style.css

```diff
@@ -482,3 +482,117 @@ button {
   font-weight: bold;
   width: auto;
 }
+
+/* contact form */
+.contact-me {
+  width: 375px;
+  height: 790px;
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  margin-top: -160px;
+  background-image: url(./images/contact-shapes.png);
+  background-repeat: no-repeat;
+  background-position-y: 9px;
+  background-position-x: 153px;
+}
+
+#contact-me {
+  width: auto;
+  height: 56px;
+  text-align: center;
+  margin-top: 78px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+#text {
+  width: 327px;
+  height: 110px;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin: 24px auto;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 53px;
+}
+
+.fields {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] I highly suggest you to use d-flex class for all your flex containers. Doing that, your code will be applied best practices for DRY principle.